swagger.yaml

```diff
@@ -1457,13 +1432,41 @@ components:
           format: uuid
           description: User's ID in Keycloak.
         attributes:
-          $ref: "#/components/schemas/Attributes"
-        severaUserId:
-          type: string
-          description: The Severa user ID associated with the Keycloak attribute.
-        severaKeyword:
+          $ref: "#/components/schemas/Attributes" 
+
+    UpdateUserAttributeResponse:
+      type: object
+      description: Response after updating user attributes.
+      properties:
+        id:
           type: string
-          description: The keyword returned from Severa API that is associated with the user.  
+          format: uuid
+          description: User's ID in Keycloak.
+        updatedAttributes:
+          $ref: "#/components/schemas/Attributes"
+        severaUser:
+          type: object
+          description: The user's information from Severa.
+          properties:
+            email:
+              type: string
+              description: The email address fetched from Severa.
+            severaUserId:
+              type: string
+              description: The Severa user ID associated with the Keycloak attribute.
+            severaKeyword:
+              type: string
+              description: The keyword returned from Severa API that is associated with the user.
+            keycloakResponse:
+              type: object
+              description: The response from the Keycloak update operation.
+              properties:
+                success:
+                  type: boolean
```

Comment:
Is this really necessary/ used? If not I would suggest removing from here and in the bugfix lambda.